README.md

```diff
@@ -89,6 +89,36 @@ You also need to disable the internal store onto the `BarComponent` with the pro
 <BarComponent disableInternalStore />
 ```
 
+## Enable search and AI assistant
+
+Search and AI assistant is now proposed by the cozy-bar. To enable it, you need to : 
+
+1. Add the data proxy in your app
+
+- cozy-dataproxy-lib must be installed in your app
+- DataProxyProvider must be added before BarProvider in your app
```

Comment:
If we forget this step, do we have a clear error?

Until now, we deployed the search only on a few app and in parallel of the search feature development, so we had control. But now we may want to do the check for better DX.

However I'm not sure if we should throw an error. For now, if no data-proxy, then we just disable the search feature (but AI assistant will work). But the check is in the case the iFrame did not load.
Maybe we can check if the dataProxyProvider is available and throw an error otherwise?

Reply:
I added a more explicit message here https://github.com/cozy/cozy-libs/pull/2751

Search disabled and no provider = ok
Search enabled and provider = ok
Search enabled and no provider = blank page but explicit error message with https://github.com/cozy/cozy-libs/pull/2751